test/functional/specs/dataElements/variable.spec.js

```diff
@@ -326,7 +326,7 @@ test.requestHooks(
   }
 );
 
-test.requestHooks(sandboxMocks.multipleWithoutDefault)(
+test.requestHooks(sandboxMocks.multipleWithoutDefault, schemasMocks.empty)(
```

Comment:
This test was flaky. An API error was showed about schemas not being loaded. This error was sometimes appearing before the test was successful, but other times it was appearing before. This test just wants to test that an error is showed when no sandbox is selected. No API error should be thrown.